vmngclient/api/templates.py

```diff
@@ -193,7 +200,7 @@ def delete(self, name: str) -> bool:
         endpoint = f"/dataservice/template/device/{template.id}"
         if template.devices_attached == 0:
             response = self.session.delete(url=endpoint)
-            return response.status_code == 200
+            return response.ok
```

Comment:
`response.ok` is not the equivalent of `response.status_code == 200` so it will also catch situation when we wanted to remove template from device but it didn't modify anything (maybe it could happen in situations when the template is not present in config). Not sure if you want to change it, its just a comment.